code/modules/reagents/newchem/medicine.dm

```diff
@@ -336,6 +337,7 @@ datum/reagent/perfluorodecalin
 	reagent_state = LIQUID
 	color = "#C8A5DC"
 	metabolization_rate = 0.2
+	addiction_chance = 20
 
```

Comment:
Hmm, this should offer a good reason to make salbut in place of perfluoro, now...